README.txt

```diff
@@ -46,6 +46,11 @@ Add this to your (child) theme’s functions.php.
 
 == Changelog ==
 
+= 1.0.7 = 
+* Set display of Additional Information Contact fields default to hidden. Change FAQ to reflect this change. Issue #26
+* Disable link to 'Meetings at this Location' feature. Issue #23
```

Comment:
it might be nice to link to the issues - on the WP changelog people aren't going to have the context that these refer to github issues